src/test/java/service/AuthServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package service;
+
+import org.example.daos.AuthDao;
+import org.example.daos.DatabaseConnector;
+import org.example.exceptions.InvalidException;
+import org.example.models.LoginRequest;
+import org.example.models.User;
+import org.example.services.AuthService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.security.Key;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+class AuthServiceTest {
+
+    AuthDao authDao = Mockito.mock(AuthDao.class);
+    Key key = Mockito.mock(Key.class);
+    LoginRequest loginRequest = Mockito.mock(LoginRequest.class);
+    User user = Mockito.mock(User.class);
```

Comment:
I don't think you need to mock these things. just create an object for each if you need to use them